docs/console/tutorials/project-infrastructure/project-configuration.md

```diff
@@ -0,0 +1,396 @@
+---
+id: project-configuration
+title: Project Infrastructure Tutorial
+sidebar_label: Project Infrastructure Tutorial
+sidebar_order: 1
+---
+
+# Project Infrastructure How To
+
+## Introduction
+The "Project Infrastructure" is a new type of project that allows centralized management of infrastructure resources.
+This type of project is exclusively dedicated to managing infrastructure resources and offers specific functionalities for creating, deploying, and managing these resources.
+
+## Creating a Project Infrastructure
+
+![IMG 1](./img/1.png)
+
+In console instances where the feature is enabled, it is now possible to create a new type of project called "Project Infrastructure".
+To create a new project, simply access the Console, enter a specific Company, and click on "Create Project".
+
+![IMG 2](./img/2.png)
+
+In the General step, under "Project type", select the "Infrastructure" option.
+
+This will change the standard project creation flow (the Environments step will no longer be present), though it will still be possible to configure the repository and its visibility.
```

Comment:
>though it will still be possible to configure the repository and its visibility.

actually there is a little difference here as you are not configuring the repository but the group (if the GitProvider supports them).

The reason for this is that there is no central repository for configuration (such as with Application Projects -the traditional ones-). On GitLab and other providers supporting folders we are only creating a folder that will hold any new repository associated with the infrastructure project you are about to create.

I don't know if this bit of information is useful here, maybe it would be most useful in the main doc (maybe it's already there, if so I'd add links to the specific sections here)